test-setup.js

```diff
@@ -0,0 +1,347 @@
+const { configure } = require('enzyme');
+
+const React = require('react');
```

Comment:
note: dropping this in via copypasta was necessary until https://github.com/airbnb/enzyme/issues/1509 gets resolved